src/main/java/unrc/dose/User.java

```diff
@@ -1,19 +1,39 @@
-/*
-* == Schema Info
-*
-* Table name: users
-*
-*  id              :integer(11)    not null, primary key
-*  username        :varchar(20)    not null,
-*  password        :varchar(20)    not null,
-*  email_address   :varchar(50)    not null,
-*  admin           BOOLEAN not null default 0
-*
-**/
+
 
 package unrc.dose;
 
 import org.javalite.activejdbc.Model;
+import org.javalite.activejdbc.LazyList;
 
 public class User extends Model {
+	static final String  PSW_RESET = "password reset";
+	static final String  PSW_NOT_RESET = "the email is not registered in the DB";
+
+	
+	 /**
+
+     * Method that returns a message if the password has been modified successfully
+     
+     * @param email the email of the user who wants to change the password
+
+     * @param pass the new password
+
+	 * @return A message if the password has been modified successfully
+	 
+	*/
+
+	public static String updatePassword(String email,String pass) {
+		LazyList<User> users = User.where("email_address = ?", email);
+		User user = new User();
+		if(users.size() > 0) {
+			user = users.get(0);
+			user.set("password",pass);
+			user.saveIt();
+			return PSW_RESET;
+		}
+		return PSW_NOT_RESET;
+	}
```

Comment:
There is no documentation attached to this method, you should always document at least all public attributes and methods. Taken into account that after a comma there should be a blank space, and between an `if` at its condition, there should also be a blank space.

The new instance in line 27, that is, `User user = new User();`, is never used, likewise, the `user` variable is only used in the `then` part of the if statement, and as such this variable should be local to this if statement.

Finally, consider that the returning value should be a `true` \ `false` value.